src/commands/server/deploy.ts

```diff
@@ -345,12 +345,6 @@ export default class Deploy extends Command {
       }
     }
 
-    if (!flags.batch && isKubernetesPlatformFamily(flags.platform) && (isDevWorkspaceEnabled(ctx) || flags['workspace-engine'] === 'dev-workspace')) {
```

Comment:
How this is related to adding resources for Cert Manager?

Reply:
Removing code duplication. Just spotted accidentally.

Reply:
https://github.com/che-incubator/chectl/blob/main/src/commands/server/deploy.ts#L342-L353